cypress/integration/admin.js

```diff
@@ -26,4 +26,22 @@ describe('Sourcebooks admin', function() {
         check.searchTaskName('[type="text"]', task[0]);
         check.visibilityCheck('[role="row"]');
     })
+
+    it('Admin creates a client', function() {
+        var task = [Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15)]
```

Comment:
please consider using key value arrays, in order to not use task[3] but instead task["email"] would be more readable and maintainable in a future

Reply:
also format this line to be like :
var task = [
     Math.random().toString(36).substring(2, 15), 
     Math.random().toString(36).substring(2, 15),
     .....
]
much easier to read that way